src/lib/Microsoft.Fx.Portability.Reports.Html/Views/_PortabilityReport.cshtml

```diff
@@ -51,7 +55,7 @@
         }
         @foreach (var invalidAssembly in reportingResult.GetAssembliesWithError())
         {
-            var fileName = Path.GetFileName(invalidAssembly);
+            var fileName = System.IO.Path.GetFileName(invalidAssembly);
```

Comment:
Was there a collision when we had the `@using System.IO` in there?

Reply:
Yup - there's a property in razor apparently that is `Path`